src/Illuminate/Auth/SessionGuard.php

```diff
@@ -183,7 +183,7 @@ protected function userFromRecaller($recaller)
      */
     protected function recaller()
     {
-        if ($recaller = $this->request->cookies->get($this->getRecallerName())) {
+        if (! is_null($this->request) && $recaller = $this->request->cookies->get($this->getRecallerName())) {
```

Comment:
Assignments inside boolean expressions should be avoided, since it leads to confusion and bugs.